tracing-api/src/main/java/com/palantir/tracing/api/TraceHttpHeaders.java

```diff
@@ -22,4 +22,16 @@
     String PARENT_SPAN_ID = "X-B3-ParentSpanId";
     String SPAN_ID = "X-B3-SpanId";
     String IS_SAMPLED = "X-B3-Sampled"; // Boolean (either “1” or “0”, can be absent)
+
+    /**
+     * Conceptually, a trace is a stack of spans. In implementation, this is actually many stacks, in many services,
+     * where a service's stack will typically contain a single parent span from a different service at the bottom,
+     * and many spans of its own above it.
+     *
+     * By communicating this deepest span id with future network calls as an 'originating' span id, this enables
+     * network-level tracing to be enabled always in a low-fidelity form, with request logs containing enough
+     * information to reconstruct a request-level trace. For service-internal tracing, the typical trace logs
+     * (with sampling) are still required.
+     */
+    String ORIGINATING_SPAN_ID = "X-OrigSpanId";
```

Comment:
Could we also make sure this header name is mentioned in the README so it's vaguely googleable?  This is an extra piece of information that we've made up that's not in ZipKin, so would be good to make sure it's super obvious!

Reply:
Capturing the naming discussion - "incoming span id" doesn't quite make sense for the first time this gets emitted as a header, because the first value sent was just made up inside the service.

Reply:
done

Reply:
Does this need to resemble a header? If we have the ability to push non-span parameters through all of the glue, it'd be nice if it were instead something like `_rootSpan` or `_origSpanId` to match with the other "special" parameters that we have.

Reply:
it is actually sent over the wire as a header, if that makes a difference?

Reply:
Ah gotcha - I was thinking this was just tracked separately on each server.